<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>517</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ISL Kinect Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/image/img/Icon.jpg</normaloff>:/image/img/Icon.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>701</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Record</string>
     </attribute>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>252</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Project Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>671</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>Logo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_framerate">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>289</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_proName">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>249</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="chb_depth">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>289</y>
        <width>83</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Depth</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setInfo">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>208</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Info</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_directory">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>209</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="framerate">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>290</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FrameRate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chb_rgb">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>249</y>
        <width>83</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>RGB</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chb_upper">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>208</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chb_skelet">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>329</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Skelet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="go_dir">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>208</y>
        <width>30</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="directory">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>209</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Project Dirctor</string>
      </property>
     </widget>
     <widget class="QLabel" name="recnumber">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>290</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>#Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_record">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>249</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_record">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>289</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QLabel" name="label_view">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>480</width>
        <height>270</height>
       </rect>
      </property>
      <property name="text">
       <string>Image View</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>517</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_previous3">
         <property name="text">
          <string>&lt;&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_previous2">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_previous1">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_next1">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_next2">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_next3">
         <property name="text">
          <string>&gt;&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="list_record">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>181</width>
        <height>270</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_about">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_exit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionCreate_New_Project"/>
    <addaction name="actionOpen_Project"/>
   </widget>
   <addaction name="menuProject"/>
  </widget>
  <action name="actionCreate_New_Project">
   <property name="text">
    <string>Create New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>go_dir</tabstop>
  <tabstop>le_directory</tabstop>
  <tabstop>le_proName</tabstop>
  <tabstop>le_framerate</tabstop>
  <tabstop>le_record</tabstop>
  <tabstop>chb_upper</tabstop>
  <tabstop>chb_rgb</tabstop>
  <tabstop>chb_depth</tabstop>
  <tabstop>chb_skelet</tabstop>
  <tabstop>setInfo</tabstop>
  <tabstop>pb_record</tabstop>
  <tabstop>pb_about</tabstop>
  <tabstop>pb_exit</tabstop>
  <tabstop>pb_previous1</tabstop>
  <tabstop>pb_next1</tabstop>
  <tabstop>pb_next2</tabstop>
  <tabstop>list_record</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pb_previous2</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
